src/v2/Components/v2/ArtworkFilter/ArtworkFilterMobileActionSheet.tsx

```diff
@@ -13,22 +21,42 @@ export const ArtworkFilterMobileActionSheet: SFC<{
 }> = ({ children, onClose }) => {
   const filterContext = useArtworkFilterContext()
 
+  const contentRef = useRef<HTMLDivElement | null>(null)
+
   // This reflects our zero state for this UI which doesn't include the keyword
   const isReset = isEqual(
     omit(filterContext.filters, "reset", "keyword"),
     initialArtworkFilterState
   )
 
+  const handleScrollToTop = () => {
+    if (!contentRef.current) return
+    contentRef.current.scrollTop = 0
+  }
+
   return (
-    <Container>
+    <ModalBase
+      onClose={onClose}
+      dialogProps={{
+        width: "100%",
+        height: "100%",
+        background: color("white100"),
+        flexDirection: "column",
+      }}
+    >
       <Header p={1}>
         <Button variant="noOutline" size="small" onClick={onClose}>
           Close
         </Button>
 
-        <Title size="3" weight="medium" textAlign="center">
-          Filter
-        </Title>
+        {/* TODO: This extraneous Flex is not necessary, Clickable (and Box) should have Flex props*/}
```

Comment:
Just updated this to utilize `Clickable` and somehow I managed to run into this https://github.com/artsy/palette/issues/704 having just opened the issue after PR-ing that component. So... There's that.